variables.tf

```diff
@@ -0,0 +1,47 @@
+variable "prefix" {
+    description = "The prefix used for all resources in this environment"
+    default = "todo"
+}
+variable "location" {
+    description = "The Azure location where all resources in this deployment should be created"
+    default = "uksouth"
+}
+
+variable "resource_group" {
+    default = "Capita1_LouisEgbufor_ProjectExercise"
+}
+
+variable "GITHUB_CLIENT_ID" { 
```

Comment:
It would be nice to add a brief description to all of these, so that their purpose is clear to future users